h/static/scripts/streamer.js

```diff
@@ -88,52 +65,25 @@ function connect($rootScope, annotationMapper, groups, session, settings) {
     session.update(message.model);
   }
 
-  function sendClientConfig () {
-    Object.keys(configMessages).forEach(function (key) {
-      if (configMessages[key]) {
-        socket.send(configMessages[key]);
-      }
-    });
-  }
-
-  /**
-   * Send a configuration message to the push notification service.
-   * Each message is associated with a key, which is used to re-send
-   * configuration data to the server in the event of a reconnection.
-   */
-  function setConfig(key, configMessage) {
-    configMessages[key] = configMessage;
-    if (socket && socket.isConnected()) {
-      socket.send(configMessage);
-    }
-  }
-
-  socket.on('open', function () {
-    sendClientConfig();
-  });
-
-  socket.on('error', function (event) {
+  function handleSocketOnError (event) {
     console.warn('Error connecting to H push notification service:', event);
 
     // In development, warn if the connection failure might be due to
     // the app's origin not having been whitelisted in the H service's config.
     //
     // Unfortunately the error event does not provide a way to get at the
     // HTTP status code for HTTP -> WS upgrade requests.
-    var websocketHost = new URL(url).hostname;
+    var websocketHost = new URL(settings.websocketUrl).hostname;
     if (['localhost', '127.0.0.1'].indexOf(websocketHost) !== -1) {
       console.warn('Check that your H service is configured to allow ' +
                    'WebSocket connections from ' + window.location.origin);
     }
-  });
+  }
 
-  socket.on('message', function (event) {
-    // Wrap message dispatches in $rootScope.$apply() so that
+  function handleSocketOnMessage (event) {
+    // wrap message dispatches in $rootScope.$apply() so that
```

Comment:
Our current code is unfortunately not consistent in places but the coding style going forwards is that comments should be written as sentences (ie. starting with a capital letter), same as the Python code.